Bottom: 9f58af7efb89068eb1a5b03f8916c657d4d6fa38
Top:    043fa6ab6b3ef538490004674f8581c179ee779b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 23:28:03 +0000

Refresh of hid-png-png-c-remove-buggy-dea

---

diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index ff8931a..892696b 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -1425,18 +1425,6 @@ png_fill_rect (hidGC gc, Coord x1, Coord y1, Coord x2, Coord y2)
   gdImageSetThickness (im, 0);
   linewidth = 0;
 
-  if (x1 > x2)
-    {
-      Coord t = x1;
-      x2 = x2;
-      x2 = t;
-    }
-  if (y1 > y2)
-    {
-      Coord t = y1;
-      y2 = y2;
-      y2 = t;
-    }
   y1 -= bloat;
   y2 += bloat;
   SWAP_IF_SOLDER (y1, y2);
